src/sqlfluff/core/rules/config_info.py

```diff
@@ -172,6 +172,11 @@
             "Optional, regex of blocked pattern which should not be used in statements."
         ),
     },
+    "match_source": {
+        "definition": (
+            "Optional, also match regex of blocked pattern before applying dbt builtins"
```

Comment:
This is a bit of a nit, but seeing as this'd apply to Jinja too, can we generalise the description to "before applying templating" or something similar?

Reply:
Done.